debian/changelog

```diff
@@ -1,3 +1,49 @@
+wiringpi (3.1-1) unstable; urgency=low
+
+  * New maintainers. Support for Pi 5 and many improvements.
+
+ -- Rafael Diniz <rafael@rhizomatica.org>  Tue, 12 Mar 2024 21:01:01 +0000
+
+wiringpi (2.70-4) unstable; urgency=low
```

Comment:
please ditch all releases unofficial releases that are not available through wiringpi/wiringpi releases on GitHub

Reply:
Done.